app/views/orders/edit.html.erb

```diff
@@ -0,0 +1,31 @@
+
+<div class="wholecart">
+  <h1> Petsy Checkout! </h1>
+  <% @cart_price = 0 %>
+  <div class="cart">
+    <% @order.orderitems.each do |orderitem| %>
+      <div class="orderitem">
+      <%= link_to (image_tag Product.find(orderitem.product_id).photo_url), product_path(orderitem.product_id) %>
+      <div class="order-info">
+      <strong><%= link_to Product.find(orderitem.product_id).name, product_path(orderitem.product_id) %></strong>
+      <br>Total Price $<%=number_with_precision orderitem.total_price, precision: 2 %>
+      <% @cart_price += orderitem.total_price %>
+      <%= form_for orderitem do |f| %>
+      <% @quantity_numbers = (1..orderitem.product.quantity).to_a %>
+        <%= f.label "Quantity" %>
+        <%= f.select 'quantity', options_for_select(@quantity_numbers, orderitem.quantity) %><br>
+        <%= f.submit "Update Quantity", class: "btn btn-info" %>
+      <% end %>
```

Comment:
Instead of iterating over ```@order.orderitems```, you can iterate over @orderitems, which is defined in the controller as ```Order.find(current_order.id).orderitems```